scout/adapter/mongo/case.py

```diff
@@ -289,18 +289,25 @@ def update_dynamic_gene_list(self, case, hgnc_symbols=None, hgnc_ids=None,
             updated_case(dict)
         """
         dynamic_gene_list = []
+        if add_only:
+            dynamic_gene_list_cursor = self.case_collection.find({ '_id': case['_id'] },
```

Comment:
From what I understand from the code if you are only interested in the first result returned from the query. Why don't you use find_one() instead of find() and the dynamic_gene_list_cursor[0] to get the first element of the cursor?